src/main/java/com/reminiscence/reminiscence/watson/WatsonController.java

```diff
@@ -0,0 +1,72 @@
+package com.reminiscence.reminiscence.watson;
+
+import com.ibm.cloud.sdk.core.service.security.IamOptions;
+import com.ibm.watson.tone_analyzer.v3.ToneAnalyzer;
+import com.ibm.watson.tone_analyzer.v3.model.DocumentAnalysis;
+import com.ibm.watson.tone_analyzer.v3.model.ToneAnalysis;
+import com.ibm.watson.tone_analyzer.v3.model.ToneOptions;
+import com.ibm.watson.tone_analyzer.v3.model.ToneScore;
+import com.reminiscence.reminiscence.journal.Entry;
+import com.reminiscence.reminiscence.journal.EntryController;
+import com.reminiscence.reminiscence.journal.EntryRepo;
+import com.sun.jndi.ldap.EntryChangeResponseControl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.security.Principal;
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
```

Comment:
This class isn't used as an endpoint, so shouldn't have been a controller.  `@Service` would have been more appropriate and would still have worked with autowiring.  (And the class name of WatsonService)

Reply:
Putting this logic into its own class was excellent, though.